doc/classes/AudioStreamPlayer3D.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioStreamPlayer3D" inherits="Spatial" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Plays 3D sound in 3D space
 	</brief_description>
 	<description>
+		Plays a sound effect with directed sound effects, dampens with distance if needed, generates effect of hearable position in space.
 	</description>
 	<tutorials>
```

Comment:
tutorials could link here:
http://docs.godotengine.org/en/latest/learning/features/audio/index.html

Reply:
ok :) how should i add that? just as the Link in the next line? never saw tutorials linked in the other docs files

Reply:
@Grosskopf As I said on Discord, place the URL inside the <tutorial> tag. It's a brand new features, fresh from reduz ^^

Reply:
thanks :D so many new channels of communication :D